src/test/java/org/alliancegenome/curation_api/crud/controllers/DiseaseAnnotationITCase.java

```diff
@@ -93,7 +95,10 @@ public void editDiseaseAnnotation() {
 
         // change subject
         BiologicalEntity newSubject = createBiologicalEntity("BO:0002", "taxon:0001");
-        editedDiseaseAnnotation.setSubject(newSubject);
+        
+        // TODO fix due to SCRUM-510 refactor
+        //editedDiseaseAnnotation.setSubject(newSubject);
+        
```

Comment:
@abecerra @markquintontulloch  do we need to check on these?

Reply:
After the conflicts are solved, I will look the edition of negated disease annotations issue in this branch and the edit integration test can be helpful